BedrockCore.cpp

```diff
@@ -17,9 +17,15 @@ bool BedrockCore::peekCommand(BedrockCommand& command) {
     command.peekCount++;
     uint64_t timeout = command.request.isSet("timeout") ? command.request.calc("timeout") : DEFAULT_TIMEOUT;
 
+    if (timeout > 2'000'000) {
+        // Old microsecond timeout. Update caller to use milliseconds. Remove this line once we no longer see this.
+        SWARN("[TYLER] old-style timeout found for command: " << command.request.methodLine);
```

Comment:
We should of used `ENSURE_BUGBOT` or something over `TYLER`  but it's not worth it to fix